### DISCUSSION TITLE
Auto HTML API setup to exclude js/css components

### DISCUSSION THREAD

**Discussion by AndreyMisnikov** [ServiceStack/Discuss#121] [Q&A]
After updating to ServiceStack 8.x, I've noticed that when I execute an API endpoint via the browser URL with a content-type of text/html, a new page called "Auto HTML API" is generated. However, it attempts to load additional JS/CSS files.

Primarily, I've configured a "Content-Security-Policy" header to specify what can or cannot be loaded, and I'm using a load balancer. The path for loading JS/CSS components is incorrect + security issues. Is there a way to set it up as it worked previously, without loading JS or CSS components? I've only found the option to disable HTML via HostConfig.EnableFeatures, but that's not the solution I'm seeking.

**Comment by mythz** [comment]
Please provide a network web console screenshot of these additional resources? There should only be references to internal resources as we've removed the [polyfil for Safari of import maps](https://github.com/ServiceStack/ServiceStack/commit/f8ac18ec2886a6d357c25e81767b1b4d36709259) in January, there shouldn't be any external references from v8.1.

Otherwise you should be able to restore the old HtmlFormat page with:

```csharp
ServiceStack.Templates.HtmlTemplates.HtmlFormatName = "HtmlFormatLegacy.html";
```

You can also remove other [Built In UIs](https://servicestack.net/auto-ui) like API Explorer with:

```csharp
Plugins.RemoveAll(x => x is UiFeature);
```
